apps/core/package.json

```diff
@@ -42,7 +42,7 @@
     "close-with-grace": "^1.2.0",
     "dotenv": "^16.3.1",
     "fastify": "^4.22.2",
-    "mongoose": "^7.5.0",
+    "mongoose": "^7.5.2",
     "ofetch": "^1.3.3",
     "pino": "^8.15.0",
     "zod": "^3.22.2"
```

Comment:
In java universe, we usually set a unique and stable version of dependencies, I am afraid of the usage of "^", how can we handle lib updates without bringing issues to our code?

Reply:
You should take a look at SemVer, it is bad practice to not use the latest. The only case where it can be a "problem" to update is when the dependencies make a SemVer major change. e.g. go from `7x` to `8x`. You should **never**
fear the update. The goal is always to reduce deps.